### PR TITLE
Added classname as a config parameter which will set svg class

### PR DESCRIPTION
**Pull request by dpiccone**
I had to introduce this for a project I am working on, maybe it can be added to the API

### CODE REVIEW

**Comment by aendrew** [comment]
:+1: Not a bad idea at all. Could use a unit test though — mind adding one? Also, when squashing your commits, please remove c3.js because that will cause merge conflicts (It generally gets built during new releases. Thanks!

**Comment by dpiccone** [comment]
I added tests and removed the minified c3 :)

File: Gruntfile.coffee [hunk 1]
```diff
@@ -89,7 +89,8 @@ module.exports = (grunt) ->
           c3:
             src: 'c3.js'
             options:
-              specs: 'spec/*-spec.js'
+                #specs: 'spec/*-spec.js'
+              specs: 'spec/core-spec.js'
```
Comment (masayuki0812) [inline Gruntfile.coffee]: Can you fix this?

**Comment by masayuki0812** [comment]
@dpiccone Thank you for the PR. LGTM basically, but can you change the config to `svg_classname`? It seems easier to understand which classname it is :)

**Comment by dpiccone** [comment]
Cleaned up Gruntfile and changed option name

**Comment by dpiccone** [comment]
Updated the config, fixed tests

**Comment by aendrew** [comment]
@dpiccone I'm working through all the PRs and figure this is a good candidate, but would you mind please squashing your commits? Thanks! RTBC otherwise! :+1:

**Comment by masayuki0812** [comment]
Thank you!!

**Pull request merged**